core/src/main/java/org/apache/accumulo/core/client/PluginEnvironment.java

```diff
@@ -55,6 +55,15 @@
      */
     String get(String key);
 
+    /**
+     * Returns all properties with a given prefix
+     *
+     * @param prefix
+     *          prefix of properties to be returned
+     * @return all properties with a given prefix
+     */
```

Comment:
I know this should probably have a `@since` tag, I'm just not sure what version to put. Please let me know and I'll add it.

Reply:
2.1.0 would be the version we would add this kind of thing to.

Reply:
Thanks, added